Network/HTTP/Base.hs

```diff
@@ -792,7 +792,7 @@ normalizeUserAgent opts req =
        _ -> replaceHeader HdrUserAgent ua req
 
 -- | @normalizeConnectionClose opts req@ sets the header @Connection: close@
--- to indicate one-shot behavior iff @normDoClose@ is @True@. i.e., it then
+-- to indicate one-shot behavior if @normDoClose@ is @True@. i.e., it then
```

Comment:
It may be a bit obscure, but "iff" is mathematical shorthand for "if and only if", so this probably wasn't a typo. Could you instead expand it out?

Reply:
new commit